app/src/main/java/com/example/cmput301f18t26/icare/Controllers/DataController.java

```diff
@@ -1,12 +1,44 @@
 package com.example.cmput301f18t26.icare.Controllers;
 
+import com.example.cmput301f18t26.icare.Problem;
+import com.example.cmput301f18t26.icare.Record;
+import java.util.ArrayList;
+
 class DataController {
-    private static final DataController ourInstance = new DataController();
+    private ArrayList<Record> records;
```

Comment:
I think we should merge these changes as is, however I don't think there there is any reason to need for storing collections of items. Instead, the data controller should grab the data, construct a new collection of objects and then ship it to the model (without storing the data in its own controller). Then the model can notify the views that the data has changed etc.

Reply:
Should we have a save to file method for offline mode?